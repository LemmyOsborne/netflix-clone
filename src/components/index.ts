export { Jumbotron } from "./Jumbotron/Jumbotron";
export { Footer } from "./Footer/Footer";