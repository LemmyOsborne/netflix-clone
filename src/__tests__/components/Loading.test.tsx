import React from "react";
import { render } from "@testing-library/react";
import { Loading } from "../../components";

describe("<Loading />", () => {
  it("renders the <Loading />", () => {
    const { container, getByTestId, getByAltText } = render(<Loading src="/images/2.png" data-testid="loading" />)
    expect(getByTestId("loading")).toBeTruthy()
    expect(getByAltText("Profile image")).toBeTruthy()
    expect(container.firstChild).toMatchSnapshot()
  })

  it("renders the <Loading.ReleaseBody />", () => {
    const { container } = render(<Loading.ReleaseBody data-testid="loading" />)
    expect(container.firstChild).toMatchSnapshot()
  })
}) 