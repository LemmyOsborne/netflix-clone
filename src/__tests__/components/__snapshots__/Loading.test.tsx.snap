// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders the <Loading /> 1`] = `
<div
  class="sc-Galmp bMWYyW"
  data-testid="loading"
  src="/images/2.png"
>
  <img
    alt="Profile image"
    class="sc-fWCJzd dhYqHi"
    src="/images/users/undefined.png"
  />
</div>
`;

exports[`<Loading /> renders the <Loading.ReleaseBody /> 1`] = `null`;
